Gemfile

```diff
@@ -21,6 +21,8 @@ gem 'jbuilder', '~> 2.0'
 # bundle exec rake doc:rails generates the API under doc/api.
 gem 'sdoc', '~> 0.4.0', group: :doc
 gem 'pg'
+gem 'carrierwave'
```

Comment:
@godfreym I think paper clip is more active than carrier wave
compare them 
[carrier wave](https://github.com/carrierwaveuploader/carrierwave)
[paperclip](https://github.com/thoughtbot/paperclip)